packages/auth/test/helpers/integration/helpers.ts

```diff
@@ -80,6 +80,33 @@ export function getTestInstance(requireEmulator = false): Auth {
   return auth;
 }
 
+export function getTestInstanceForServerApp(
+  serverApp: FirebaseServerApp,
+  requireEmulator = false
```

Comment:
I didn't see a usage where `requireEmulator` was set to true, is this in anticipation of running required emulator tests with serverApp in the future?

Reply:
Honestly, I was just reusing the logic for `getTestInstance` above, but you're right, we never define `requiresEmulator` to true. I'll remove it.

Reply:
Done.